modules/v2x/fusion/libs/common/base/box.h

```diff
@@ -0,0 +1,185 @@
+/******************************************************************************
```

Comment:
This file is exactly the same as modules/perception/base/box.h. Why not just reuse it?